tools/pbstate/src/pbstate.c

```diff
@@ -354,3 +354,19 @@ int pbstate_set_system_verified(pbstate_system_t system)
     }
     return pbstate_commit();
 }
+
+int pbstate_read_board_reg(unsigned int index, uint32_t *value)
+{
+    if (index > (NO_OF_BOARD_REGS - 1))
```

Comment:
To allow for (easier) later extension with more board_regs, I would at least consider numbering them with index == 0 as the _last_ register, and then count backwards.

Reply:
fixed